_posts/2022-01-27-partner-hidora.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Partner Highlight: Enjoy OpenSearch one click installation with Hidora"
+authors:
+  - piellick 
+date: 2022-01-27
+categories:
+  - partners
+twittercard:
+  description: "Hidora is an expert in providing custom solutions for all sizes of businesses. Hidora Cloud Hosting Solutions are just what you need if you want an efficient, secure and scalable cloud hosting solution that is tailored to your needs. We offer different plans that are scalable according to your specific requirements so you can find one that suits you best. Our PaaS help developers to host their applications in few minutes with DevOps philosophy."
+---
+
+Who we are
+==========
+
+[Hidora](https://hidora.io) is an expert in providing custom solutions for all sizes of businesses. Hidora Cloud Hosting Solutions are just what you need if you want an efficient, secure and scalable cloud hosting solution that is tailored to your needs. We offer different plans that are scalable according to your specific requirements so you can find one that suits you best. Our PaaS help developers to host their applications in few minutes with DevOps philosophy.
+
+Why choosing Hidora for your OpenSearch hosting ?
+=================================================
+
+With our partner Jelastic, we have integrated the deployment of the OpenSearch stack in every detail. We are currently able to offer you version 1.2.3. Also, redeploy functionality provides a simple update process. You can scale your OpenSearch cluster, extend the storage, add nodes in few seconds.
+
+
+
+One click installation with the Topology wizard !
+-------------------------------------------------
+
+Take advantage of our customization interface to define the architecture of your cluster:
+
+![Hidora Topology wizard Cluster Dashboard](/assets/media/blog-images/2022-01-27-hidora-partner/image01.png){: .img-fluid}
```

Comment:
I think the filename is `image1.png` not `image01.png`.

Reply:
yes sorry for the mistake.